src/main/java/Dobby.java

```diff
@@ -0,0 +1,138 @@
+import java.util.Scanner;
+
+public class Dobby {
+
+    private static final String DASH_LINE = "____________________________________________________________";
+    private static final int MAX_LIST_SIZE = 100;
+    private static final Task[] taskList = new Task[MAX_LIST_SIZE];
+
+    private static int listSize = 0;
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        String line;
+
+        printWelcomeMessage();
+
+        while (true){
+            line = in.nextLine().trim();
+            if (line.equalsIgnoreCase("bye")){
+                break;
+            }
+            processCommand(line);
+        }
+
+        printGoodbyeMessage();
+    }
+
+    private static void processCommand(String line) {
+        if (line.equals("list")) {
+            printList();
+        } else if (line.startsWith("mark ")) {
+            markTaskAsDone(line);
+        } else if (line.startsWith("unmark ")) {
+            unmarkTaskAsDone(line);
+        } else {
+            addTask(line);
+        }
+    }
+
+    private static void printWelcomeMessage(){
```

Comment:
There is a missing space for your method.
```suggestion
    private static void printWelcomeMessage() {
```